Jenkinsfile

```diff
@@ -25,10 +25,6 @@ if (addParameters && !params.BUILD_TEE) {
     return
 }
 
-buildGramine(
-    buildInfo: buildInfo,
-    dockerfileDir: 'gramine'
-)
```

Comment:
You can remove one of the empty lines here.
Globally, we want to avoid keeping successive empty lines in our files to keep a good readability.